compass/landice/tests/ismip6_run/ismip6_ais_proj2300/namelist.sealevel.template

```diff
@@ -0,0 +1,66 @@
+&time_config
+    itersl = 1
+    starttime = 2015
+    dt1 = 5
+
+/
+&model_resolution
+    norder = 512
+    nglv = "{{ nglv }}"
+
+/
+&io_directory
+    inputfolder_ice = "{{ slm_input_ice }}"
+    inputfolder = "{{ slm_input_others }}"
+    planetfolder = "{{ slm_input_earth }}"
+    gridfolder = "{{ slm_input_others }}"
+    outputfolder = OUTPUT_SLM/
+    outputfolder_ice = ICELOAD_SLM/
+    folder_coupled = ''
+
+/
+&file_format
+    ext =''
+    fType_in = 'text'
+    fType_out = 'both'
+
+/
+&file_name
+    planetmodel   = 'prem_512.l60K2C.sum18p6.dum19p2.tz19p4.lm22'
```

Comment:
Can we add a cfg option to set up either WAIS or typical Earth structure?

Reply:
@matthewhoffman, given that no one earth model can be a representative of a region or the globe, I was hesitant to add a config option with names like 'WAIS' or 'typical'. Instead, I made a config option with which the users define the filename of the earth model they want use.